docs/reference/chart-title.md

```diff
@@ -15,6 +15,12 @@ chts=<font_color>,<font_size>
 
 - **`<font_color>`** The title color, in [RRGGBB[AA] hexadecimal format](/reference/color-format). Default color is black.
 - **`<font_size>`** Font size of the title, in points.
+- **`<opt_alignment>`** Not implemented yet.
+- **`<opt_font_family>`** [Optional] one of the [supported open-source font](/reference/chart-font/#supported-open-source-fonts). Don't forget to checkout the [font gallery](https://fonts.google.com/).
```

Comment:
`Don't forget to checkout the [font gallery](https://fonts.google.com/).` à supprimer, ça sera déjà dans la doc "chart-font"